.travis.yml

```diff
@@ -11,6 +11,9 @@ matrix:
     env:
     - TESTS=true
     - COVERAGE=true
+  - go: "1.11.1"
```

Comment:
at this rate we're going to hit the 1 hour mark for CI, we should do some research into smarter CI or combine some steps into 1.

Reply:
most steps = more parallelization. The real issue is that Travis is throttling us on the number of concurrent steps.